### ISSUE TITLE
RCB_ERROR An error occured while loading the filter settings: 'xbmcgui.ControlList' object has no attribute 'setLabel'

### ISSUE THREAD

**Issue by loggin** [maloep/romcollectionbrowser#404]
Help, finally bit the bullet an upgraded to kodi 18 (think I was still on 16 if not 15) so have had to update RCB and now can't import any collections, I'm getting errors saying about setLabel not existing, is it trying to call old methods ? Have I got something lingering from the upgrade ?

RCB_ERROR An error occured while loading the filter settings: 'xbmcgui.ControlList' object has no attribute 'setLabel'

I did uninstall RCB before installing this latest version

**Comment by maloep** [comment]
Latest version of RCB is 2.2.3 - more info here: https://forum.kodi.tv/showthread.php?tid=70115&pid=2757114#pid2757114). This should work with Kodi 18 without bigger issues.

If you still have issues, please attach a kodi log file as described here: https://kodi.wiki/view/Log_file/Easy

**Comment by loggin** [comment]
I've tried uninstalling and reinstalling and still getting the setLabel error, there's also some control id mapping errors, it just doesn't appear to be trying to scan my folders at all

[kodi.txt](https://github.com/maloep/romcollectionbrowser/files/3509170/kodi.txt)

**Comment by maloep** [comment]
I think this is a skin issue. Aeon MQ 8 is known not to be working with RCB atm. Please try another skin or delete all RCB files from your "skin.aeonmq8" xml directory (the file "script-Rom_Collection_Browser-main.xml" and all files that start with "script-RCB").

**Comment by loggin** [comment]
@maloep thanks, I removed the xml files and now it's importing my collections, any plans to get Aeon MQ 8 compatible with RCB ?

**Comment by Kupo91** [comment]
I'm suprised that they still implement their non working skin files. I already told them about this issue in Aeon MQ7 and whenever I asked them for a fix Wanilton said he will have a look on it (but he never did).

In the end I asked someone to build the panel view for RCB (thats what I cared the most about this aeon skin files). The thing is even if they fix their files (what they won't do) its possible that the files stop working again with the next RCB update. Thats what I noticed the last updates. With each update it stopped working more and more.

However if we have this design integrated in RCB as skin as we have for Estuary, Embuary, Confluence and Arctic Zephyr we dont have this issue.
